lib/Jsrt/ChakraCore.h

```diff
@@ -104,6 +104,53 @@ typedef JsErrorCode(CHAKRA_CALLBACK * FetchImportedModuleFromScriptCallBack)(_In
 /// </returns>
 typedef JsErrorCode(CHAKRA_CALLBACK * NotifyModuleReadyCallback)(_In_opt_ JsModuleRecord referencingModule, _In_opt_ JsValueRef exceptionVar);
 
+/// <summary>
+///     A structure containing information about a native function callback.
+/// </summary>
+struct JsNativeFunctionInfo
+{
+    JsValueRef thisArg;
+    JsValueRef newTargetArg;
+    bool isConstructCall;
```

Comment:
Is it worth documenting how these work? e.g. if `isConstructCall` is true, then `newTargetArg` will be non-null, otherwise it will be null?

Reply:
I think `newTargetArg` cannot be non-null unless `isConstructCall` is true but not sure if the example above is always true. Thinking about it, `new.target` should really be undefined and not null, I believe.